cli/src/cmd/forge/script/transaction.rs

```diff
@@ -41,6 +41,8 @@ pub struct TransactionWithMetadata {
     pub transaction: TypedTransaction,
     pub additional_contracts: Vec<AdditionalContract>,
     pub is_fixed_gas_limit: bool,
+    pub bundle_block: Option<U64>,
+    pub bundle_gas: Option<U256>,
```

Comment:
Might be overkill, but I do think having the bundle details in a separate struct and then wrapped in one `Option` would be nicer because it seems they are both always checked together and also both must always be present.